.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: required
+services:
+  - docker
+
+env:
+  COMPOSE_VERSION: 1.9.0
+
+before_install:
+ - curl -L https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+ - chmod +x docker-compose
+ - sudo mv docker-compose /usr/local/bin
+
+script:
+  - docker-compose up -d cassandra elasticsearch redis
```

Comment:
It would be better to not start the whole stack each times. The API of the frontend tests are all mocked and should be fast (tests are run each time a new commit is pushed in a branch/PR).